tests/test_answer_quality.py

```diff
@@ -0,0 +1,125 @@
+import json
+import pytest
+import logging
+from logging import StreamHandler
+import litellm
+import pandas as pd
+from jinja2 import Environment, FileSystemLoader
+
+from agents.ansari import Ansari
+
+LOGGER = logging.getLogger(__name__)
```

Comment:
should this be done in this way or as recommended in the python tutorial: https://docs.python.org/3/howto/logging.html